front/src/common/ReleaseInformations/LicenseAttributions.tsx

```diff
@@ -2,57 +2,95 @@ import { useTranslation } from 'react-i18next';
 
 import attributionsLicenses from './json/licenses.json';
 
-type attributionsLicensesType = {
+type AttributionLicense = {
   name: string;
   version: string;
   copyright: string;
   publisher: string;
   url?: string;
-  licenses: string;
+  license: string;
 };
 
-const LicenseAttributions = () => {
-  const { t } = useTranslation('home/navbar');
-
-  // mglTraffic is not a lib from package.json, so we add it manually
-  const mglTrafficAttribution: attributionsLicensesType = {
-    licenses: 'CC-BY-NC-SA 3.0',
+// Libs which are not in the package.json, so we add it statically
+const DATA_SOURCES: AttributionLicense[] = [
+  {
+    license: '',
```

Comment:
it seems there is a license: https://geoservices.ign.fr/cgu-licences

Reply:
Done